invenio_rdm_migrator/streams/runner.py

```diff
@@ -50,14 +50,16 @@ def __init__(self, stream_definitions, config_filepath):
         )
         STATE.initialized_state(self.state)
         # set up secret keys
-        STATE.VALUES.add(
-            "old_secret_key",
-            {"value": bytes(config.get("old_secret_key"), "utf-8")},
-        )
-        STATE.VALUES.add(
-            "new_secret_key",
-            {"value": bytes(config.get("new_secret_key"), "utf-8")},
-        )
+        if not STATE.VALUES.get("old_secret_key"):
```

Comment:
IIUC, this means that on the first run they are set. Then in the following runs, they are not. If we are checking for existence, maybe we should also check that both do, because otherwise, it will fail. In addition, this means that if you make a mistake on the first configuration and it is persisted you cannot change it unless you manually delete it?

Reply:
Agree, both should be set in all cases (even if they're there). I'll make the change so that we call `.add()`/`.update` accordingly.